openshift/e2e-common.sh

```diff
@@ -108,7 +108,7 @@ spec:
     spec:
       containers:
       - name: zipkin
-        image: docker.io/openzipkin/zipkin:2.13.0
+        image: docker.io/openzipkin/zipkin:latest
```

Comment:
Doesn't zipkin have a mirror image that isn't hit by Docker Hub limits?

Reply:
That is a question for zipkin.... I am not really up for maintaining a different storage for that....

FWIW, the openzipkin has a two year old quay.io repo.... but that is no longer maintained, hence _NOT_ using that :wink: 

If their community want to change something regarding Docker Hub limits, it's really an issue for them.